src/gui/advancedsettings.h

```diff
@@ -59,7 +59,7 @@ private slots:
     template <typename T> void addRow(int row, const QString &rowText, T *widget);
 
     QLabel labelQbtLink, labelLibtorrentLink;
-    QSpinBox spinBoxAsyncIOThreads, spinBoxCache, spinBoxSaveResumeDataInterval, spinBoxOutgoingPortsMin, spinBoxOutgoingPortsMax, spinBoxListRefresh, spinBoxMaxHalfOpen,
+    QSpinBox spinBoxCheckingMemUsage, spinBoxAsyncIOThreads, spinBoxCache, spinBoxSaveResumeDataInterval, spinBoxOutgoingPortsMin, spinBoxOutgoingPortsMax, spinBoxListRefresh, spinBoxMaxHalfOpen,
```

Comment:
If you're placing the new option after "async IO threads", please do it here as well.

Reply:
Ok, will do.